static/bidder-params/fwssp.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "FWSSP Adapter Params",
+  "description": "A schema which validates params accepted by the FWSSP adapter",
+  "type": "object",
+
+  "properties": {
+    "custom_site_section_id": {
+      "type": ["string"],
```

Comment:
type is `string` and not an array of strings. Use directly:
`"type": "string",`